src/backend/game_config.ts

```diff
@@ -267,7 +267,8 @@ class GameConfigV0 extends GameConfig {
       beforeLaunchScriptPath,
       afterLaunchScriptPath,
       gamescope,
-      verboseLogs
+      verboseLogs,
+      advertiseAvxForRosetta: isMac && wineVersion.type === 'toolkit'
```

Comment:
This will cause the globally-set value for this setting to be ignored. Ideally handle it like the other options (set a default value in GlobalConfig, then just destructure the property here)

Reply:
Cool, will change.